daemon/logger/jsonfilelog/jsonfilelog.go

```diff
@@ -33,7 +34,12 @@ func New(filename string) (logger.Logger, error) {
 func (l *JSONFileLogger) Log(msg *logger.Message) error {
 	l.mu.Lock()
 	defer l.mu.Unlock()
-	err := (&jsonlog.JSONLog{Log: string(msg.Line) + "\n", Stream: msg.Source, Created: msg.Timestamp}).MarshalJSONBuf(l.buf)
+	timestamp, err := timeutils.FastMarshalJSON(msg.Timestamp)
+	if err != nil {
+		return err
+	}
+	line := append(msg.Line, '\n')
```

Comment:
You can use append right in line below without creating variable. Don't know if it's worth it.